src/Countries/Mexico.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\Carbon;
+use Carbon\CarbonImmutable;
+use Carbon\CarbonPeriod;
+
+class Netherlands extends Country
+{
+    public function countryCode(): string
+    {
+        return 'mx';
+    }
+
+    /** @return array<string, string|CarbonImmutable> */
+    protected function allHolidays(int $year): array
+    {
+
+        $natalicioBenitoJuarez = new CarbonImmutable(sprintf("third monday of march %s", $year));
+        $promulgacionConstitucion = new CarbonImmutable(sprintf("first monday of february %s", $year));
+        $revolucionMexicana = CarbonImmutable::createFromFormat('Y-m-d', "{$year}-11-20")->setTimezone('America/Mexico_City');
```

Comment:
**Wrong date!** As per [mexican government website](https://www.gob.mx/profedet/articulos/sabes-cuales-son-los-dias-de-descanso-obligatorio-para-este-2024) "El día de la revolución mexicana" is the third monday of november